cdk/lib/__snapshots__/prism-access.test.ts.snap

```diff
@@ -105,9 +97,7 @@ Object {
           },
           Object {
             "Key": "Stage",
-            "Value": Object {
-              "Ref": "Stage",
-            },
+            "Value": "INFRA",
```

Comment:
This is the (singleton) stack set, so taking the opportunity to move to the `INFRA` stage.